static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -79,18 +81,18 @@ const autoUpdate = (opts) => {
         });
     };
 
-    const injectNewBlocks = (newBlocks) => {
+    const injectNewBlocks = (newBlocks, userInteraction) => {
         // Clean up blocks before insertion
         const resultHtml = $.create(`<div>${newBlocks}</div>`)[0];
         let elementsToAdd;
 
         fastdom.mutate(() => {
             bonzo(resultHtml.children).addClass('autoupdate--hidden');
             elementsToAdd = Array.from(resultHtml.children);
+            if (userInteraction) $liveblogBody.empty()
```

Comment:
preeeeety sure we don't need this parameter `userInteraction` - remove from everywhere

Reply:
@abeddow91

Reply:
Removed.